docker-compose.yml

```diff
@@ -37,19 +37,21 @@ services:
     container_name: msmt-fta-fmea-converter
 
   fta-fmea:
-    image: ghcr.io/kbss-cvut/fta-fmea-ui/tool:pr-82
+    image: ghcr.io/kbss-cvut/fta-fmea-ui:latest
     environment:
       API_URL: ${API_URL:-http://localhost:8090/fta-fmea-server}
       ADMIN_REGISTRATION_ONLY: ${ADMIN_REGISTRATION_ONLY:-false}
     depends_on:
       - fta-fmea-server
 
   fta-fmea-server:
-    image: ghcr.io/kbss-cvut/fta-fmea/tool:pr-24
+    image: ghcr.io/kbss-cvut/fta-fmea:latest
     depends_on:
       - graphdb
     environment:
       REPOSITORY_URL: ${REPOSITORY_URL:-http://graphdb:7200/repositories/fta-fmea}
+    volumes:
+      - fta-fmea-server:/tmp
```

Comment:
Co se tady do /tmp uklada? /tmp mi nezni moc perzistentne :)

Reply:
Netusim, delal jsem to podle termitu

Reply:
Ale tipuju, ze to tam nejspis neni ani potreba

Reply:
neni potreba pokud nedelas nejake soubory ktere je potreba mit persistovane

Reply:
@Matthew-Kulich prosim vzdy tady odpoved, at nemusim zjistovat (I kdyz je tam Outdated tak je to mnohokrat slozite zjistit, hlavne kdyz je tam vic souboru)

Reply:
Dobre. 

Removed.